composites/AlgoliaSearch/SearchBar.js

```diff
@@ -137,7 +183,12 @@ class SearchBar extends React.Component {
 						autoCorrect="off"
 						autoCapitalize="off"
 						spellCheck="false"
+						placeholder={ placeholderText }
```

Comment:
Do we want the possibility to pass the placeholder text as a prop?

Reply:
No 🙂